packages/plugins/foundry-contracts/ChugSplash.sol

```diff
@@ -340,24 +338,16 @@ contract ChugSplash is Script, Test, DefaultCreate3, ChugSplashManagerEvents, Ch
             );
         }
 
-        if (_route == ProposalRoute.RELAY || _route == ProposalRoute.REMOTE_EXECUTION) {
-            ffiCommitToIPFS(_deploymentId);
-        }
-
-        if (_route == ProposalRoute.RELAY) {
-            ffiRelayProposal(_deploymentId);
-        } else {
```

Comment:
Since the standalone propose task is only ever used against live networks and only sends transactions using our backend, I don't see a strong reason to maintain a Solidity implementation of this extra path. I believe the propose task can be largely shared between the hardhat and foundry implementations. 

I will verify this in a separate task once we're done with the deploy task. If it is necessary to port more logic for the propose task, then I will do it then.

Reply:
Yeah, I believe that's what I did, let me know if you meant something else.

Reply:
Yeah my bad, didn't realize you already removed them